api/MediaStreamTrack.json

```
@@ -1119,6 +1119,40 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "screenPixelRatio": {
+          "__compat": {
+            "description": "Return object contains `screenPixelRatio`",
+            "spec_url": "https://w3c.github.io/mediacapture-screen-share/#dom-mediatracksettings-screenpixelratio",
+            "support": {
+              "chrome": {
+                "version_added": "136"
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false
```

There is already a Firefox bug:

```suggestion
                "version_added": false
                "impl_url": "https://bugzil.la/1965499"
```

---

Yes, I wondered if there was (or should be) some standardised & documented way for a subfeature to say "returned object contains property x". ALso wonder if it would be helpful to link to the page for the property.